pkg/router/route_group.go

```diff
@@ -367,9 +367,11 @@ func (rg *RouteGroup) keepAliveLoop(interval time.Duration) {
 				continue
 			}
 
-			if err := rg.sendKeepAlive(); err != nil {
-				rg.logger.Warnf("Failed to send keepalive: %v", err)
-			}
+			go func() {
+				if err := rg.sendKeepAlive(); err != nil {
```

Comment:
I also feel like `sendKeepAlive` should range through `rg.tps` and use `rg.wrotePacketAsync`.